snake.py

```diff
@@ -1,34 +1,40 @@
 import math
 import random
 import pygame
-from pygame import *
 import tkinter as tk
-from tkinter import messagebox

```

Comment:
This is another duplicate import. import tkinter as tk already allows you to use tk.messagebox. The import on line 6 allows you to use messagebox without the preceding tk.

You can add this import back in, but it is mostly redundant